e2e/test/scenarios/visualizations-tabular/column-shortcuts/combine-column-shortcut.cy.spec.ts

```diff
@@ -55,6 +55,26 @@ describeWithSnowplow("scenarios > visualizations > combine shortcut", () => {
       database_id: SAMPLE_DB_ID,
     });
   });
+
+  it("should not show the shortcut button when the table is raw", function () {
```

Comment:
nit: can't it be a unit test?

Reply:
There are currently no unit tests for this component and there is a lot of setup that might need to happen that I'm not sure I'm comfortable enough with to provide.

Reply:
I mean this test looks exactly like a unit test, but I understand why you did like this